source/agora/consensus/EnrollmentManager.d

```diff
@@ -439,6 +450,17 @@ public class EnrollmentManager
         return makeEnrollment(utxo, key, cycle[height], cycle_length);
     }
 
+    /// Ditto
+    version (unittest) public static Enrollment makeEnrollment (
+        in Hash utxo, in KeyPair key, in Hash cycle_seed, in Height seed_pos,
+        in Height height, uint cycle_length)
+        @trusted nothrow
+    {
+        // Generate the random seed to use
+        auto cycle = PreImageCycle(cycle_seed, seed_pos, cycle_length);
+        return makeEnrollment(utxo, key, cycle[height], cycle_length);
+    }
```

Comment:
I think you can just replace the previous function, I don't see why we would want to use the production value and generate 5M PIs in tests.
Also I would change the order of parameters to match other overloads:
```
    version (unittest) public static Enrollment makeEnrollment (
        in Hash utxo, in KeyPair key, in Height height, uint cycle_length,
        in Hash cycle_seed, in Height seed_pos)
        @trusted nothrow
```
It's usually pretty confusing (and dangerous) to have overloads which have swapped parameters.

Reply:
In `agora.consensus.data.genesis`, the previous function should be used. So I leave the function as it is. But I changed the order of parameters to match other overloads.